README.md

```diff
@@ -41,6 +41,16 @@ OR
 
   Then remember to call `:PluginInstall`.
 
+OR
+
+- Use [vim-plug]() by adding to your `.vimrc` in your plugin section:
```

Comment:
```suggestion
- Use [vim-plug][] by adding to your `.vimrc` in your plugin section:
```

Reply:
Can you explain that please @k-takata ?

Reply:
Please check the file on GitHub: https://github.com/editorconfig/editorconfig-vim/blob/add2baa345d3c5c33b0689d148df5acf4e045e6e/README.md
You may see that the link to vim-plug is broken. See also the lines 31 and 36.